tck-dist/src/main/starter/suite.xml

```diff
@@ -17,6 +17,9 @@
 -->
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 
+<!-- 
+    This suite file is for testing Jakarta Concurrency against the Jakarta Full Profile.
+ -->
 <suite name="jakarta-concurrency" verbose="2" configfailurepolicy="continue">
     <test name="jakarta-concurrency.tck">
         <packages>
```

Comment:
This file will also need to be updated to exclude the Web profile duplicated tests that I will be working on today.  Otherwise, we will be adding tests to the Full profile run that were not there in the original release.

I will include that in my PR and It shouldn't hold up this one from being merged.